flink-queryable-state/flink-queryable-state-client-java/src/test/java/org/apache/flink/queryablestate/client/state/ImmutableAggregatingStateTest.java

```diff
@@ -53,20 +54,28 @@ public void setUp() throws Exception {
         aggrState = ImmutableAggregatingState.createState(aggrStateDesc, out.toByteArray());
     }
 
-    @Test(expected = UnsupportedOperationException.class)
-    public void testUpdate() throws Exception {
-        String value = aggrState.get();
-        assertEquals("42", value);
+    @Test
+    void testUpdate() {
+        assertThatThrownBy(
+                        () -> {
+                            String value = aggrState.get();
+                            assertEquals("42", value);
 
-        aggrState.add(54L);
+                            aggrState.add(54L);
```

Comment:
We should make the code block of `assertThatThrownBy` as small as possible, ideally it only contains codes that will throw errors.